src/runtime_src/core/tools/common/ReportHost.cpp

```diff
@@ -70,16 +70,18 @@ ReportHost::writeReport(const xrt_core::device * _pDevice,
     _output << boost::format("  %-20s : %s\n") % "Release" % _pt.get<std::string>("host.os.release");
     _output << boost::format("  %-20s : %s\n") % "Version" % _pt.get<std::string>("host.os.version");
     _output << boost::format("  %-20s : %s\n") % "Machine" % _pt.get<std::string>("host.os.machine");
+    _output << boost::format("  %-20s : %s\n") % "CPU Cores" % _pt.get<std::string>("host.os.cores");
+    _output << boost::format("  %-20s : %s MB\n") % "Memory" % _pt.get<std::string>("host.os.memory");

```

Comment:
O.K. - I now see that we are dealing with MBytes.

Reply:
Being that the original data is coming in as a hex string, and it is desired to report it as MBs, the following code could be of assistance:
```
static const uint64_t BYTES_TO_MEGABYTES = 0x1000000;
_output << boost::format("  %-20s : 0x%x MB\n") % "Memory" % strtoll(_pt.get<std::string>("host.os.memory").c_str(),nullptr,16)/BYTES_TO_MEGABYTES);
```

Reply:
Thanks for the suggestion -- done!